scripts/variantstore/wdl/extract/populate_alt_allele_table.py

```diff
@@ -23,7 +23,7 @@ def execute_with_retry(label, sql):
             job_config = bigquery.QueryJobConfig(labels=job_labels)
             query = client.query(sql, job_config=job_config)
 
-            print(f"STARTING - {label}")
+            print(f"STARTING - {label} (jobid: {query.job_id})")
```

Comment:
since you are adding logging... can you log the bytes scanned when the query completes so we can estimate costs easier?

Reply:
added to the "COMPLETED" message 👍🏻